loadtest/main.go

```diff
@@ -288,12 +333,17 @@ func getLastHeight() int {
 	return height
 }
 
+func setup() {
```

Comment:
Do you still need this if we have `GetValidators` already?

Reply:
good catch, forgot to remove this